FlashDevelop/MainForm.cs

```diff
@@ -1032,6 +1031,8 @@ private void InitializeComponents()
             this.LocationChanged += new EventHandler(this.OnMainFormLocationChange);
             this.GotFocus += new EventHandler(this.OnMainFormGotFocus);
             this.Resize += new EventHandler(this.OnMainFormResize);
+
+            ScintillaManager.ConfigurationLoaded += ApplyAllSettings;
```

Comment:
I checked it, but I'm not completely sure at the moment, I need some rest heh. That ApplyAllSettings call is an old one, I'd also say it can be removed, but I don't know if there is some reasoning behind it, because the old code although less OOP had a similar result.